test/EVMHost.cpp

```diff
@@ -174,10 +174,13 @@ void EVMHost::newTransactionFrame()
 {
 	// Clear EIP-2929 account access indicator
 	recorded_account_accesses.clear();
-	// Clear EIP-2929 storage access indicator
+
 	for (auto& [address, account]: accounts)
 		for (auto& [slot, value]: account.storage)
-			value.access_status = EVMC_ACCESS_COLD;
+		{
+			value.access_status = EVMC_ACCESS_COLD; // Clear EIP-2929 storage access indicator
+			value.dirty = false;					// Clear EIP-2200 dirty slot flag
```

Comment:
See https://github.com/ethereum/evmc/blob/v8.0.0/include/evmc/mocked_host.hpp#L24

Reply:
This has been there since the beginning and we probably never supported it properly.